src/bp/nlu-core/entities/custom-entity-extractor.ts

```diff
@@ -74,6 +74,24 @@ function computeStructuralScore(a: string[], b: string[]): number {
   return Math.sqrt(final_charset_score * token_qty_score * token_size_score)
 }
 
+// returns list entities having cached results in one array and those without results in another
+function splitModels(
```

Comment:
I know I wrote this one, you can rename it to a complexe name if you find it not specific enough

Reply:
like `splitByCacheMissOrHit` ?